### DISCUSSION TITLE
Whats the current progress of this API?

### DISCUSSION THREAD

**Discussion by jasonmbrown** [VeroFess/PalWorld-Server-Unoffical-Api#36] [General]
This sounds fairly promising, I have a few questions though. Will this be a standalone server, or will it hook into the user hosted server? What is the current progress (aka: Without delving into the current code base what stage of development is this project at)?

Are the current plans to be entirely server side, or will a client side api be used to create more complex modifications?

**Comment by VeroFess** [comment]
Currently, this project is in a state of suspension of development, mainly for two reasons:
First of all, I have been busy with work recently and do not have much time to deal with this project.
Secondly, plans are underway to use a more general approach to binary code merging rather than simply using hooks.
